_extensions/quarto-svgbob/quarto-svgbob.lua

```diff
@@ -42,15 +44,36 @@ function Render(elem)
 	return nil
 end
 
+function InsertSvgLatex(svg_data)
+	local name = 'assets/'
+	local isok, errstr, errcode = os.rename(name, name)
+	if isok == nil then
+		if errcode ~= 13 then
+			os.execute("mkdir assets")
+		end
+	end
```

Comment:
Is it worth copying over the `os.exists` and `os.mkdir` functions (with a credit to the source) from here?

https://github.com/mokeyish/obsidian-enhancing-export/blob/16cdb17ef673e822e03e6d270aa33b28079774cc/lua/polyfill.lua#L25-L47

Reply:
I will check it....